tools/topology/topology2/CMakeLists.txt

```diff
@@ -1,4 +1,8 @@
 # SPDX-License-Identifier: BSD-3-Clause
+
+# 3.17 required: foreach(<loop_var>... IN ZIP_LISTS <lists>)
+cmake_minimum_required(VERSION 3.17)
```

Comment:
Even if you put it here it will still block people trying to build say the logger with CMake 3.16. Just bump this line to 3.99 and try to build the logger and you will see.

So I have mixed feelings about this. It looks local to topology2 but it's not. OK let's say it's useful to have here for 1) documentation purposes, 2) in case someone decouples all the CMakefiles...